libvirt/resource_libvirt_domain_test.go

```diff
@@ -1767,3 +1767,58 @@ func testAccCheckLibvirtDomainDestroy(s *terraform.State) error {
 	}
 	return nil
 }
+func TestAccLibvirtDomain_FwCfgName(t *testing.T) {
+	var domain libvirt.Domain
+	var volume libvirt.StorageVol
+	randomDomainName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	randomFwCfgName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	randomPoolName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	randomPoolPath := "/tmp/terraform-provider-libvirt-pool-" + randomPoolName
+	randomIgnitionName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	var config = fmt.Sprintf(`
+	data "ignition_systemd_unit" "acceptance-test-systemd" {
+		name    = "example.service"
+		content = "[Service]\nType=oneshot\nExecStart=/usr/bin/echo Hello World\n\n[Install]\nWantedBy=multi-user.target"
+	}
+
+	data "ignition_config" "acceptance-test-config" {
+		systemd = [
+		"${data.ignition_systemd_unit.acceptance-test-systemd.id}",
+		]
+	}
+
+    resource "libvirt_pool" "%s" {
+        name = "%s"
+        type = "dir"
+        path = "%s"
+    }
+
+	resource "libvirt_ignition" "%s" {
+		name    = "ignition"
+		content = "${data.ignition_config.acceptance-test-config.rendered}"
+        pool    = "${libvirt_pool.%s.name}"
+	}
+
+	resource "libvirt_domain" "%s" {
+		name            = "terraform-test-domain"
```

Comment:
change this name with a unique name. for moment this is coped pasted. we can randomize it.

Good would be that your randomize it also in coreos but is not releated to this